modules/ocf/manifests/packages/docker.pp

```diff
@@ -10,16 +10,13 @@
 #
 class ocf::packages::docker($admin_group = undef,
                             $autoclean = true,
-                            $image_max_age = '24h') {
+                            $image_max_age = '24h',
+                            $container_max_age = '24h',
+                            $prune_volumes = true) {
   class { 'ocf::packages::docker::apt':
     stage => first,
   }
 
-  package {
-    'docker.io':
-      ensure  => purged;
-  }
```

Comment:
The `docker-ce` package `Replaces: docker-engine`, so we don't need the purge rule this time.